ss-tproxy

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
In any case, there will be absolute paths in shebang. Even if `/usr/bin/env bash` is used, there is a similar problem: if the path of the env command is not `/usr/bin/env`, it will not work. So I don't think it's necessary to change it. When it really doesn't work, it's just to modify the first line.